tests/test_parsers.py

```diff
@@ -33,3 +33,10 @@ def test_court_type_crs(report, state):
     assert parser.matches is not None, "Regex match failed"
     assert parser.matches == {"Superior": "Yes"}
     assert report["General"]["Superior"] == "Yes"
+
+
+def test_additional_offenses(report, state):
+    string = """ ***    Additional offenses exist. To see a complete breakdown, view full case detail. * * * """
+    matches = parsers.AdditionalOffenses(report, state).match(string)
+    assert matches is not None, "Regex match failed"
+    assert matches["value"]
```

Comment:
Can you assert that the `matches["value"]` equals `Additional offenses exist. To see a complete breakdown, view full case detail.` since we currently always expect it to be that value?

Reply:
I was actually planning to just make it return True for the value. You can see this on line 57 of parser/lines.py. I figured that logically it's really a boolean and the language we want to display on the page is more of a front-end concern. What do you think?